code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -101,6 +100,9 @@
 
 /mob/living/simple_animal/hostile/retaliate/goat/AttackingTarget()
 	. = ..()
+	if(. && isliving(target))
+		var/mob/living/L = target
+		L.visible_message("<span class='warning'>[src] rams [L]!</span>")

```

Comment:
so you're making it produce a message before checking to see if the person it's "attacking" is a podperson?